control/joy_controller/README.md

```diff
@@ -42,23 +42,23 @@
 | `max_backward_velocity`   | double | absolute max velocity to go backward                                                                               |
 | `backward_accel_ratio`    | double | ratio to calculate deceleration (commanded acceleration is -ratio \* operation)                                    |
 
-## P65 Joystick Key Map
```

Comment:
Please do not remove `P65` and newly add `DS4`.

Reply:
Done